src/Terminus.php

```diff
@@ -201,6 +203,8 @@ private function addBuiltInCommandsAndHooks()
     private function configureContainer(ContainerInterface $container)
     {
         // Add the services.
+        $container->add(Client::class);
+        $container->add(HttpRequest::class);
```

Comment:
And I'm using the container (instead of instantiating and injecting) because Request uses a new client and http_request for each call of `->request()`. Maybe we can inject something reusable to make the interface simpler for that helper.